sample-xml/README.md

```diff
@@ -0,0 +1,51 @@
+# Android Templates
+
+- Our optimized Android templates used in our android projects
+
```

Comment:
Should we remove this? 😁

Reply:
@Tuubz Remove it or update it to say it's a sample generated from our template showing XML layouts? 😄

Reply:
> a sample generated from our template

Even better 👍

Reply:
@Tuubz How about `test` package?

Reply:
@lydiasama You plan to remove it? 💭

Reply:
@Tuubz Since we intended to keep the sample to be simple, so I would ask about shall we add the unit test too? If not, then shall we remove it to keep the repository size to be small? 🤔  What do you guys think? @nimblehq/android-chapter

Reply:
I'm fine to keep it, could be nice to add more tests for sample code in the future 🙏

Reply:
Okie. cc @chornerman

Reply:
Got it 😄

Reply:
What about: 

```
# Android XML Sample

A sample project generated from our template using XML layouts
```